internal/configuration/config.template.yml

```diff
@@ -1,10 +1,13 @@
-###############################################################
-#                   Authelia configuration                    #
-###############################################################
+---
```

Comment:
I'm not going to recomment all the suggestions from the root `config.template.yml` but just a note to ensure you copy the changes over when/if accepted.